XMonad/WorkspaceLayout/Util.hs

```diff
@@ -0,0 +1,11 @@
+module XMonad.WorkspaceLayout.Util where
+
+(!%) :: [a] -> Int -> a
+xs !% n = xs !! (n `mod` length xs)
```

Comment:
Is this a common operator? If so, let's add it to `XMonad.Prelude`

Reply:
I wouldn't be surprised if it's pretty common, but as far as I know it doesn't have any standard name